nevergrad/benchmark/experiments.py

```diff
@@ -613,6 +613,7 @@ def yabbob(
     split: bool = False,
     tiny: bool = False,
     tuning: bool = False,
+    reduced: bool = False,
```

Comment:
reduction_factor? because otherzise tiny and reduced are similar

Reply:
i've used benchmark_reduction because reduction_factor would suggest an int (which would be a bad idea as we need a number which is coprime with the sizes of loops...).

Reply:
I'd rather have an int and a check that the int is correct then, for now these multiple parameters do not make sense to me :s

Reply:
You mean:
- extracting all lists (one per loop)
- checking coprimality 
Feasible, but the code will not be compact and I'm not sure this will clarify anything.

Reply:
no, just use `reduction_factor: int` and then:
```python
assert reduction_factor in [1, 13, 17]  # needs to be a cofactor
functions = functions[::reduction_factor]
```
I guess that's enough and much simpler and do not add parameters which should be mutually exclusive

Reply:
Yes right this simplies, and this also simplifies yatinybbob and future stuff.